k8s/bases/swe-d2cc/swe-d2cc-minio-automatic-purge.yaml

```diff
@@ -0,0 +1,44 @@
+kind: CronJob
+apiVersion: batch/v1
+metadata:
+  name: swe-d2cc-boundary-minio-automatic-purge
+  labels:
+    app: gridcapa
+    component: boundary-minio-automatic-purge
+    region: swe
+    process: d2cc
+spec:
+  schedule: "0 0 * * *"
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 3
+  successfulJobsHistoryLimit: 3
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+            - name: swe-d2cc-boundary-minio-automatic-purge
+              image: minio-mc
+              command: ["/bin/sh","-c"]
+              args:
+                - rm -r --force --older-than 366d minio/gridcapa/SWE/D2CC/BOUNDARIES/;
+                  rm -r --force --older-than 7d minio/gridcapa/SWE/D2CC/CGMs/;
+                  rm -r --force --older-than 7d minio/gridcapa/SWE/D2CC/CRACs/;
+                  rm -r --force --older-than 7d minio/gridcapa/SWE/D2CC/GLSKs/;
+                  rm -r --force --older-than 7d minio/gridcapa/SWE/D2CC/XIIDM/;
+                  find minio/gridcapa/SWE/D2CC/20* -type f -mtime +7 -delete;
+                  find minio/gridcapa/SWE/D2CC/20* -type d -empty -delete;
```

Comment:
![image](https://user-images.githubusercontent.com/42716685/221245424-7d442332-dc1a-4476-8663-c5efff14c566.png)

I got this error in Azure dev environment